content/docs/ref/pr.md

```diff
@@ -45,7 +44,8 @@ The `--auto-merge` option enables
 [auto–merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request)
```

Comment:
the line above this should say "The `--merge`/`--rebase`/`--squash` options [...]"